### ISSUE TITLE
How can I configure conventionalcommits properties without having the preset installed into my own projects package.json?

### ISSUE THREAD

**Issue by jeacott1** [TriPSs/conventional-changelog-action#210]
Hi,
 as I understand it, if I want to customise IssuePrefixes etc I need to create a new file and reference it with config-file-path:
eg:
```
'use strict'
const config = require('conventional-changelog-conventionalcommits');

module.exports = config({
    "issuePrefixes": [""XXXX-"],
    "issueUrlFormat": "https://myproject.atlassian.net/browse/{{prefix}}{{id}}"
})
```

this disables the 'preset:' option and the conventional-changelog-conventionalcommits is no longer pulled automatically.

if I run npm install it eg: 
```
run: npm install conventional-changelog-conventionalcommits
```
This fixes the missing dependency issue, but it also includes and commits the dependency in MY project. this is totally undesirable.

How can I set the issuePrefixes and issueUrlFormat without having this injected into my own package.json?
```
"conventional-changelog-conventionalcommits": "^5.0.0" 
```

when I just use the 'preset: conventionalcommits' option this problem doesnt occur.

thanks

**Comment by TriPSs** [comment]
Why not just add the dependency to your project?

**Comment by gamunu** [comment]
> Why not just add the dependency to your project?

There are other languages, for example, GoLang.

**Comment by TriPSs** [comment]
This is actually solved with #247, please update to the latest version. We now default include `conventionalcommits` preset.

**Comment by gamunu** [comment]
awesome thank you!
